library/src/main/java/com/schibsted/spain/barista/assertion/BaristaVisibilityAssertions.kt

```diff
@@ -70,4 +71,18 @@ object BaristaVisibilityAssertions{
         onView(withText(text)).check(matches(not(isDisplayed())))
     }
 
+    @JvmStatic
+    fun assertContains(text: String) {
+        assertDisplayed(withContainsText(text))
```

Comment:
Hey! No need to create this matcher from scratch. Hamcrest+Espresso already have this functionality. 
It's as simple as using the composed matcher: `withText(containsString("string"))`.

To avoid reinventing the wheel, please remove this custom matcher and use existing ones.

Reply:
@Sloy I've tried to use `containsString`, but it returns `Matcher<String>` and it needs a `Matcher<View>` mmmm

Reply:
I'm saying combine `withText()` with `containsString()`. 

`withText()` takes a `Matcher<String>` and returns a `Matcher<View>`.

Reply:
Cool

Reply:
Done @Sloy